fastlane-plugin-github_job_status/README.md

```diff
@@ -30,9 +30,10 @@ github_job_status(
 )
 ```
 
+* `token` can be obtained in GitHub for an owner (To do this, go to settings/Personal access tokens. Generate a new token and be sure to enable `repo:status`.)
  * `state` must be pending, success, error, or failure
- * `token` can be obtained in GitHub for an owner (To do this, go to settings/Personal access tokens. Generate a new token and be sure to enable `repo:status`.)
- * `job_name`, `description` and `build_url` are optional, but all other parameters are required
+ * `sha`, is optional and default value is last commit hash get thanks to lane [last_git_commit](https://docs.fastlane.tools/actions/#last_git_commit)
+ *`job_name`, `description` and `build_url` are optionals, but all other parameters are required
```

Comment:
Also, optionals->optional

And take out "but all other parameters are required", since sha is now optional